client/src/components/NavTabs.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from "classnames";
+
+/**
+ * Render navigation tabs on CMS
+ *
+ * @param tabs
+ * @returns {JSX.Element}
+ * @constructor
+ */
+const NavTabs = ({ tabs, tabComponent: TabComponent }) => {
+  const navTabs = tabs.map((tab, index) => {
+
+    const tabClass = classNames(
+      "nav-tabs__tab-item",
+    );
+
+    const tabKey = `${tab.title}-${index}`;
+    return (
+      <TabComponent className={tabClass} role="tab" {...tab}>
```

Comment:
I removed the `loadComponent` statement here. Basically it should be the job of the parent component to provide a valid component.